sql/driver/wire.go

```diff
@@ -26,43 +29,34 @@ const (
 )
 
 func (d Datum) String() string {
-	if d.BoolVal != nil {
-		if *d.BoolVal {
-			return "true"
-		}
-		return "false"
-	}
-	if d.IntVal != nil {
-		return strconv.FormatInt(*d.IntVal, 10)
-	}
-	if d.FloatVal != nil {
-		return strconv.FormatFloat(*d.FloatVal, 'g', -1, 64)
-	}
-	if d.BytesVal != nil {
-		return string(d.BytesVal)
+	val := d.GetValue()
```

Comment:
Yeah, GetValue() kinda sucks, but it's a closer interface to what we'll soon be using: https://github.com/golang/protobuf/commit/ef197e1a5f817199945aa5c661145b2c5f4b963f#diff-fe7ebfedd1e0c5013b2415edc0c10b17R157